sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -858,6 +858,15 @@ class QueryExecutionErrorsSuite
           "any enclosing class nor any supertype")),
       sqlState = "XX000")
   }
+
+  test("SPARK-42845: merge unsupported by aggregate window function") {
+    checkError(
+      intercept[SparkUnsupportedOperationException]{
+        RowNumber().mergeExpressions
```

Comment:
Can you trigger the error from user space (by SQL statement, for instance)?

Reply:
I can't find a method to trigger this error_class using SQL statement. This error_class are only thrown by "mergeExpressions" field of AggregateWindowFunction. But this field isn't called anywhere in all aggregate window function. Maybe we can change it to internal error?